helix/principles/file-structure/modules.rst

```diff
@@ -7,13 +7,18 @@ Each module has the following structure:
 
     /…
         /[Module Name]     // Module root folder, named after the module (without prefixes)
-            /code          // Houses the main code for the module, for example the Visual Studio project with the module business logic or views.
+            /website       // Houses the website code for the module, for example the Visual Studio project with the website business logic or views.
             /serialization // Contains the serialized data from Sitecore, for example templates, content or security data
             /tests         // Unit or other test types for the module
+            /engine        // Houses the code for the plugin that runs on the Commerce Engine
 
 .. admonition:: Habitat Example
 
     .. figure:: _static/image17.png
 
         Figure: Solution and module structure on disk
 
+Note that not all modules will contain all of the elements seen above. Some of
```

Comment:
Maybe emphasize this earlier to underline that all the folders are optional dependent on module responsibility.

Reply:
Agreed, moved to above and made more prominent.